stripe/resource_stripe_plan.go

```diff
@@ -35,12 +36,14 @@ func resourceStripePlan() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 				Computed: true,
+				ConflictsWith: []string{"amount_decimal"},
```

Comment:
Maybe as to why I added this.

I had wrongfully assumed that I have to use both. It somehow worked, but basically forced re-creating plans constantly. Setting one or the other fixed it, so I figured, might as well "error", especially since the API docs say it should be one or the other.

Reply:
Got it, makes total sense :-)